extension/data/frame/frame.js

```diff
@@ -27,29 +10,20 @@ var getDuration = function (aEntry) {
   let keys = ['max_slice_pause', 'duration',
               'MaxPause', 'max_pause',
               'Total', 'TotalTime', 'total_time'];
-
-  for each (let key in keys) {
-    if (key in aEntry) {
-      return aEntry[key];
-    }
+  let key;
+
+  if (keys.some(aElement => {
+    key = aElement;
+    return key in aEntry;
+  }))
+  {
+    return aEntry[key];
   };
```

Comment:
Those brackets are related to the if condition so move up the opening bracket to the end of the former line. Also no need for the final semicolon.